controllers/datadogagent/object/configmap/configmap.go

```diff
@@ -45,3 +46,33 @@ func BuildConfiguration(owner metav1.Object, configDataPointer *string, configMa
 	}
 	return configMap, nil
 }
+
+// BuildConfigMapMulti use to generate a configmap containing configuration (yaml) or check code (python).
+// Use boolean `validate` to validate against yaml.
+func BuildConfigMapMulti(owner metav1.Object, configDataMap map[string]string, configMapName string, validate bool) (*corev1.ConfigMap, error) {
```

Comment:
I'm not sure we need to provide the `owner` because this info should be added by the dependency store automatically.